tasks/php.js

```diff
@@ -19,30 +20,41 @@ module.exports = function (grunt) {
 			}, function (res) {
 				var statusCodeType = Number(res.statusCode.toString()[0]);
 				if ([2, 3, 4].indexOf(statusCodeType) !== -1) {
-					return cb();
+					if (!serverStarted) {
+						serverStarted = true;
+						done(true);
+					}
+					return;
 				} else if (statusCodeType === 5) {
 					grunt.fail.warn(
 						'Server docroot returned 500-level response. Please check ' +
 						'your configuration for possible errors.'
 					);
-					return cb();
+					if (!serverStarted) {
+						serverStarted = true;
+						done(true);
+					}
```

Comment:
I don't understand - why a new line and no ;  ?

There was no blank line and a ; when it was `return cb();`, and there is none in binVersionCheck either.